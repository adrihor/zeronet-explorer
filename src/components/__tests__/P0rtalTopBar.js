import React from 'react'
import renderer from 'react-test-renderer'

import Component from '../P0rtalTopBar'

// FIXME This is the only solution we have to fix this issue
//       https://github.com/itinance/react-native-fs/issues/404
jest.mock('react-native-fs', () => {})

test('renders correctly', () => {
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
})
