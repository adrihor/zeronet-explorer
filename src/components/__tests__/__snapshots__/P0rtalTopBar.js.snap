// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(245, 245, 245, 1.0)",
      "flexDirection": "row",
      "height": 56,
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      Object {
        "justifyContent": "center",
        "width": 175,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "black",
          "fontSize": 26,
        }
      }
    >
      P0rtal
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "width": 100,
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "margin": 7,
          },
          Object {
            "borderRadius": 20,
            "height": 36,
            "width": 36,
          },
          false,
          Object {
            "alignItems": "center",
            "backgroundColor": "#6f07ef",
            "justifyContent": "center",
          },
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 16,
            },
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
            ],
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "margin": 7,
          },
          Object {
            "borderRadius": 20,
            "height": 36,
            "width": 36,
          },
          false,
          Object {
            "alignItems": "center",
            "backgroundColor": "#6f07ef",
            "justifyContent": "center",
          },
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "white",
              "fontSize": 16,
            },
            Array [
              Object {
                "backgroundColor": "transparent",
              },
              undefined,
            ],
            Object {
              "fontFamily": "FontAwesome",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
</View>
`;
